src/renderers/shared/hooks/ReactHostOperationHistoryHook.js

```diff
@@ -7,22 +7,33 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactHostOperationHistoryHook
+ * @flow
  */
 
 'use strict';
 
-var history = [];
+import type { DebugID } from 'ReactInstanceType';
+
+export type Operation = {instanceID: DebugID} & (
+  {type: 'mount', payload: string} |
+  {type: 'insert child', payload: {toIndex: number, content: string}} |
+  {type: 'move child', payload: {fromIndex: number, toIndex: number}} |
+  {type: 'replace children', payload: string} |
+  {type: 'replace text', payload: string} |
+  {type: 'replace with', payload: string} |
+  {type: 'update styles', payload: any /* Style Object */} |
```

Comment:
I'm still unsure how to handle things that are not properly typed yet. One one side `mixed` is good because it gives some kind of safety as flow is going to try and guess what the type is, on the other side, having `any` is great because it shows up as untyped in the flow coverage tool in Nuclide.

So far, I've chosen to leave everything that isn't properly typed as `any`. This way either something is fully and correctly typed and we can have faith in it, or it's not and shows up as completely untyped.

Please let me know what you think about it :)